app/views/manageBusinesses/manage/ConfirmReportingMethod.scala.html

```diff
@@ -29,30 +29,44 @@
     govukWarningText : GovukWarningText,
     hmrcPageHeading: HmrcPageHeading,
     p: p,
-    govukInsetText: GovukInsetText
+    govukInsetText: GovukInsetText,
+    govukRadios : GovukRadios,
+    errorSummary: ErrorSummary
 )
 
-@(backUrl: String,
+@(
+    backUrl: String,
     postAction: Call,
     isAgent: Boolean,
     taxYearStartYear: String,
     taxYearEndYear: String,
     newReportingMethod: String,
     isCurrentTaxYear: Boolean,
-    incomeSourceType: IncomeSourceType)(implicit messages: Messages, user: auth.MtdItUser[_])
+    incomeSourceType: IncomeSourceType,
+    form: Form[_],
+    optInOutContentFSEnabled: Boolean
+)(implicit messages: Messages, user: auth.MtdItUser[_])
 
-@prefix = @{ "manageBusinesses.manage.propertyReportingMethod" }
+@prefixOld = @{ "manageBusinesses.manage.propertyReportingMethod" }
+@prefixNew = @{ "manageBusinesses.manage.propertyReportingMethod.new" }
 
 @getMessage(key: String, args: String*) = @{
-    messages(s"$prefix.$key", args: _*)
+    if(optInOutContentFSEnabled){
+        messages(s"$prefixNew.$key", args: _*)
+    }else{
+        messages(s"$prefixOld.$key", args: _*)
+    }
 }
 
 @mainTemplate(pageTitle = getMessage(s"heading.$newReportingMethod"),
     backUrl = Some(backUrl),
     isAgent = isAgent,
     btaNavPartial = user.btaNavPartial) {
 
-    @formWithCSRF(action = postAction, Symbol("id") -> s"$prefix-form") {
+
+    @if(optInOutContentFSEnabled) {
+
+        @errorSummary(form)
```

Comment:
The error when you don't selet anything should when clicked take you to one of the radios